packages/pino-log-enricher/lib/createFormatter.js

```diff
@@ -39,6 +39,24 @@ module.exports = function createFormatter(newrelic) {
         }
         return obj
       }
+    },
+    hooks: {
+      logMethod(inputArgs, method, level) {
+        const config = newrelic.shim.agent.config
+
+        // TODO: update the peerdep on the New Relic repo and thereby
+        // remove check for existence of application_logging config item
+        if (config.application_logging && config.application_logging.metrics.enabled) {
```

Comment:
Small paranoia: if config is `{ application_logging: {} }` then this will fail. I would recommend adding another layer to the `&&` checks:

`if (config.application_logging && config.application_logging.metrics && config.application_logging.metrics.enabled) { ... }`

Reply:
We can't trust our own API? Can our customers stomp over it?

Reply:
Nevermind, this should be fine. Resolving this thread...